staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -254,13 +254,13 @@ func getIPFromInterface(intfName string, forFamily AddressFamily, nw networkInte
 		if err != nil {
 			return nil, err
 		}
-		klog.V(4).Infof("Interface %q has %d addresses :%v.", intfName, len(addrs), addrs)
+		klog.V(4).InfoS("Interface's addresses", "interfaceName", intfName, "addressesNum", len(addrs), "addresses", addrs)
```

Comment:
s/addressesNum/lenOfAddresses

Reply:
I think the original Infof is more readable.

Reply:
Maybe “addressesNum” is  better to tell what the number of addresses rather than lenth of address.